logviewer/src/main/res/menu/menu_categories.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item android:id="@+id/actionSearch"
+        android:title="@string/action_search"
+        android:icon="@android:drawable/ic_menu_search"
+        app:showAsAction="always"
+        app:actionViewClass="androidx.appcompat.widget.SearchView" />
+
+    <item
+        app:showAsAction="ifRoom"
```

Comment:
The presentation shouldn't know anything about our data layer implementation detail. This should be easily replacable after the other comments are accounted for.

Reply:
Didn't quite whats this is referring to, hope it's fixed already